src/lib.rs

```diff
@@ -641,10 +648,14 @@ impl<T: Counter> Histogram<T> {
             lowest_discernible_value: low,
             significant_value_digits: sigfig,
 
-            bucket_count: 0, // set by establish_size below
+            // set by cover() below
+            bucket_count: 0,
             sub_bucket_count: sub_bucket_count,
 
-            leading_zero_count_base: 0, // set below, needs establish_size
+
+            // Establish leading_zero_count_base, used in bucket_index_of() fast path:
+            // subtract the bits that would be used by the largest value in bucket 0.
+            leading_zero_count_base: 64 - unit_magnitude - sub_bucket_count_magnitude,
```

Comment:
Now  that `sub_bucket_count_magnitude` is simply ``sub_bucket_half_count_magnitude + 1`, we can use it here.